### ISSUE TITLE
Feature Request: Images in cards

### ISSUE THREAD

**Issue by PhirosWolf** [tema6120/ForgetMeNot#66]
Hello!

First of all, I really love your app, free, open-source, good design, ..., it's really great!

After having watched [this video](https://youtu.be/rhgwIhB58PA) by Veritasium and by having thought about it for a little while, I can only think it would only improve the memorization of the flashcards if we could have images with or without additional text.

I don't know how the export to CSV would work with this feature (maybe by including the URL of the image if it was provided as an URL in the first place, or just removing the images from the CSV) or for the card inversion feature (you could guess a picture from a set of pictures though).

Unfortunately, I don't know Kotlin. Otherwise, I would have helped you on this project.

I hope this is not difficult to implement.

**Comment by tema6120** [comment]
Yeah. I consider supporting images & other type of media files. Most likely the media cards would be imported as .zip file with special structure.

Showing pictures is not really difficult thing to implement. The point is my app is too tailored for the text format. Adding support for media files will affect almost all parts of application. I home sometime in the future I will find time, bring my power together and finally implement this highly-requested feature.

**Comment by DanielKonopka** [comment]
@tema6120 Any update on this, dear dev?
